hr_payroll_extended/README.rst

```diff
@@ -0,0 +1,48 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Payroll Commissions and Reinbursements
+=========================

```

Comment:
Please include a line before and after the document title.
Line must be as long as the title